modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/GeneNameDescProjection_conf.pm

```diff
@@ -390,4 +390,11 @@ sub pipeline_analyses {
   ];
 }
 
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        %{$self->SUPER::resource_classes},  # inherit 'default' from the parent class
+        '1Gb_job'    => { 'LSF' => [' -q production-rh74 -C0 -M1000 -R"select[mem>1000] rusage[mem=1000]"'] },
```

Comment:
```suggestion
        '1Gb_job'    => { 'LSF' => [' -q production -M 1000 '] },
```

Reply:
Dumb me, I just restored the initial values, commit / push, then... I did the same update.